tests/ext/app/test_ext_app.cpp

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#define CATCH_CONFIG_MAIN
+
+#include <chrono>
+#include <mutex>
+#include <thread>
+
+#include "catch_wrapper.hpp"
+#include "common_tests.h"
+#include "service_helper.h"
+#include "test_ext_app.h"
+
+#include "../../sample/ebpf.h"
+
+#define SAMPLE_PATH ""
+
+#define EBPF_CORE_DRIVER_BINARY_NAME L"ebpfcore.sys"
+#define EBPF_CORE_DRIVER_NAME L"ebpfcore"
+
+#define EBPF_EXTENSION_DRIVER_BINARY_NAME L"test_ebpf_ext.sys"
+#define EBPF_EXTENSION_DRIVER_NAME L"testebpfext"
+
+#define EBPF_SERVICE_BINARY_NAME L"ebpfsvc.exe"
+#define EBPF_SERVICE_NAME L"ebpfsvc"
+
+static service_install_helper
+    _ebpf_core_driver_helper(EBPF_CORE_DRIVER_NAME, EBPF_CORE_DRIVER_BINARY_NAME, SERVICE_KERNEL_DRIVER);
+
+static service_install_helper
+    _ebpf_extension_driver_helper(EBPF_EXTENSION_DRIVER_NAME, EBPF_EXTENSION_DRIVER_BINARY_NAME, SERVICE_KERNEL_DRIVER);
+
+static service_install_helper
+    _ebpf_service_helper(EBPF_SERVICE_NAME, EBPF_SERVICE_BINARY_NAME, SERVICE_WIN32_OWN_PROCESS);
+
+static ebpf_result_t
+_program_load_helper(
+    _In_ const char* file_name,
+    _In_ const ebpf_program_type_t* program_type,
+    ebpf_execution_type_t execution_type,
+    _Outptr_ struct _ebpf_object** object,
+    _Out_ fd_t* program_fd)
+{
+    ebpf_result_t result;
+    const char* log_buffer = nullptr;
+    result = ebpf_program_load(file_name, program_type, nullptr, execution_type, object, program_fd, &log_buffer);
+    printf("Logbuffer: %s\n", log_buffer);
```

Comment:
Another printf to replace with a REQUIRE.  (Either in this PR or in a separate one if you file a github issue.)

Reply:
Removed printf. See https://github.com/microsoft/ebpf-for-windows/pull/336#discussion_r668854535